doc/user/bmp.rst

```diff
@@ -57,6 +57,51 @@ The `BMP` implementation in FRR has the following properties:
 
 - monitoring peers with :rfc:`5549` extended next-hops has not been tested.
 
+RFC 7854
+========
+Unsupported features (non exhaustive):
+  - Per-Peer Header
+    
+    - Peer Type Flag
+    - Peer Distingsher
+  
+  - Peer Up
+  
+    - Reason codes (according to TODO comments in code)
+
+Peer Type Flag and Peer Distinguisher can be implemented easily using RFC 9069's base code.
+    
+RFC 9069
+========
+Everything that isn't listed here is implemented and should be working.
+Unsupported features (should be exhaustive):
+
+- Per-Peer Header
+
+  - Timestamp
+
+    - set to 0
+    - value is now saved `struct bgp_path_info -> locrib_uptime`
+    - needs testing
+
+- Peer Up/Down
+
+  - VRF/Table Name TLV
+  
+    - code for TLV exists
+    - need better RFC understanding
+
+- Peer Down Only
+
+  - Reason code (bc not supported in RFC 7854 either)
+
+- Statistics Report
+
+  - Stat Type = 8: (64-bit Gauge) Number of routes in Loc-RIB.
+  - Stat Type = 10: Number of routes in per-AFI/SAFI Loc-RIB. The value is 
+    structured as: 2-byte AFI, 1-byte SAFI, followed by a 64-bit Gauge.
+ 
+   
```

Comment:
I would prefer the TODO list of BMP in a separate file in doc/developer/bmp.rst

Reply:
> * rebase to origin/master (especially for [b74844e](https://github.com/FRRouting/frr/commit/b74844edb21a267f06ff3113274166649217c9e2))
i think the branch is already rebased on top of (at least) e7e1561 of origin/master. also commit cd6c698 includes the changes between b74844e and e7e1561. sorry i made such a mess, but the changes are there 😅 

> opened question: instead of using LOC_RIB flag, what about checking about peer value (if == bgpd->peer_self) ? This idea may not bring clarity on the code and may not be good.
i believe using the LOC_RIB flag matches the rest of the architecture better and makes the code clearer imo. also (iirc) some Loc Rib paths are not self originated / redistributed and still have the peer attribute set to the peer ( != peer_self)

> Overall, this code looks good. Because there is no topotests, manual testing is needed. Do you have some references on tests done ?
thanks. i can attach my configuration if you want. it contains 7 routers (CE1 PE1 P1 PE2 CE2 + 2 collectors (one for VRF ipv4 bmp traffic, the other for DEFAULT VRF ipv4 + vpnv4 traffic)) so it will be a bit lengthy

Reply:
sure, will do

Reply:
see [dd6bf8a](https://github.com/FRRouting/frr/commit/dd6bf8ade441007923e83f078cbdedbad5a273ff)